scripts/ci_harbor_artifact_count.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# This script is created to determine if a harbor image exists in the repository
+# It will print the number of artifacts that exist for a particualar image.
+LOG_FILE="harbor_check.log"
+echo "CI Harbor Check Log File" > "$LOG_FILE"
+
```

Comment:
Should we be using `set -u` flag for `nounset causes the shell to treat unset variables as an error and exit.`

Reply:
Nice catch.